data_diff/__init__.py

```diff
@@ -14,7 +14,7 @@
 def connect_to_table(
     db_info: Union[str, dict],
     table_name: Union[DbPath, str],
-    key_columns: str = ("id",),
+    key_columns: Tuple[str] = ("id",),
```

Comment:
thanks, good catch. You're reminding me that we need to add static type checking to this codebase